test/unit/operations/abstract_command.test.ts

```diff
@@ -0,0 +1,58 @@
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+
+import {
+  type Callback,
+  type ClientSession,
+  CommandOperation,
+  type CommandOperationOptions,
+  type Document,
+  type OperationParent,
+  Server,
+  ServerDescription
+} from '../../mongodb';
+import { topologyWithPlaceholderClient } from '../../tools/utils';
+
+class ConcreteCommand<T> extends CommandOperation<T> {
+  constructor(parent?: OperationParent, options?: CommandOperationOptions) {
+    super(parent, options);
+  }
+
+  async executeCommand(
+    server: Server,
+    session: ClientSession | undefined,
+    cmd: Document
+  ): Promise<any> {
+    return super.executeCommand(server, session, cmd);
+  }
+
+  protected executeCallback(
+    server: Server,
+    session: ClientSession | undefined,
+    callback: Callback<any>
+  ) {
+    return [server, session, callback];
+  }
+}
+
+describe('class CommandOperation', () => {
+  let server: Server;
+  beforeEach(() => {
+    server = new Server(
+      topologyWithPlaceholderClient([], {} as any),
+      new ServerDescription('a:1'),
+      {} as any
+    );
+  });
+
+  context('when a server is created', () => {
+    it('calls server.commandAsync when executeCommand is invoked', async () => {
+      const operation = new ConcreteCommand<any>();
+      const serverSpy = sinon.stub(server, 'commandAsync');
+      const commandPromise = operation.executeCommand(server, undefined, { ping: 1 });
+      expect(commandPromise).to.be.instanceOf(Promise<any>);
```

Comment:
```suggestion
      expect(commandPromise).to.be.instanceOf(Promise);
```

TIL that syntax was valid, but no need to parameterize the instanceof check